app/lib/efile/nj/nj1040_calculator.rb

```diff
@@ -260,19 +260,20 @@ def calculate_line_15
       end
 
       def calculate_line_20a
-        applicable_1099rs = @intake.state_file1099_rs.select do |state_file_1099r|
-          state_file_1099r.state_specific_followup.present? && state_file_1099r.state_specific_followup.income_source_none?
-        end
-
-        applicable_1099rs.sum(&:taxable_amount).round
+        non_military_1099rs.sum(&:taxable_amount).round
```

Comment:
This is a small refactor to use the existing method in the calculator here rather than redoing the logic. This will get updated again in my fast-follow refactor for both this line and line_20b to use the helper's `@non_military_1099rs` instead